llvm/lib/Transforms/Instrumentation/MemProfUse.cpp

```diff
@@ -62,6 +62,11 @@ static cl::opt<bool>
                                      "context in this module's profiles"),
                             cl::Hidden, cl::init(false));
 
+static cl::opt<bool>
+    PrintFunctionGuids("memprof-function-guids",
```

Comment:
Is the useful only by itself or typically when `memprof-print-match-info` is enabled? I'm wondering if we can just condition on the print match info instead of adding a new flag.